timerMgr.js

```diff
@@ -85,6 +86,18 @@ class TimerMgr {
   hasTimer(key) {
     return key in this.timers;
   }
+	
+  /**
+   * @param {*} key name of the timer
+   * @returns {number} of seconds left in the timer function
+   */
+  currentTimerLength(key) {
+    if (key in this.timers) {
+		return this.timers[key]['when'].toEpochSecond() - time.toZDT().toEpochSecond();
```

Comment:
For consistency, this should return a `time.Duration` instead of number of seconds.

```
    return time.Duration.between(this.timers[key]['when'], time.toZDT());
...
    return time.Duration.ofSeconds(0);
```

Reply:
I hate to do this but it just occurred to me, does it make sense to return 0 if the timer doesn't exist? Shouldn't it return `null`? I can go either way. Is there some use case where getting a 0 length duration makes sense for a Timer that doesn't exist?

As for the change to between, your approach is better. I wasn't thinking. `now` should clearly be the first arguments. If the Timer has already passed we'll still get a negative duration but I think that's desireable. "How long since this timer ran?"